config/default.yml

```diff
@@ -1,4 +1,8 @@
 AllCops:
+  Excludes:
+    - 'bin/**'
+    - 'config/**'
```

Comment:
I don't think the `config` directory should be excluded as developers put their own code in here, especially in `config/initializers`.

Reply:
Would excluding everything but `config/initializers` work? I'm trying to avoid outcomes like https://github.com/alphagov/imminence/pull/514/commits/2ba500827d44833109f523a4fa3ed6a697ee2220.

Reply:
I think we'd need to allow `config/environments`, `config/initializers`, `config/application.rb`, `config/routes.rb`, `config/schedule.rb` (and I may have forgotten some more). In that imminence example, do you have a link to the code which was causing problems? I'd imagine/hope most of the framework generated things in `config` are well linted.

Reply:
I'll allow all of `config` then, as that's pretty much it.

The things causing problems were actually `script/` and `lib/tasks/cucumber.rake` (which had the "edit at your peril" line ☠️), so I must have misremembered. [Gist of its output with the cop re-enabled](https://gist.github.com/issyl0/2deed4ae849b5c85d58a1ff79d04f031).

Reply:
Do we want to exclude `script/`?